runtime/util/ConfigurationSerializer.cpp

```diff
@@ -327,6 +329,33 @@ void serializeConfigurationInternal(
   }
 }
 
+void serializeConfigurations(
+    const char *filename, std::unordered_set<block *, HashBlock, KEq> results) {
+  FILE *file = fopen(filename, "w");
+  auto state = serialization_state();
+
+  writer w = {file, nullptr};
+  ssize_t size = results.size();
```

Comment:
Minor style nitpick: `auto`

Reply:
writer can't be inferred by `auto`

```
 error: deduced conflicting types ('_IO_FILE *' vs 'nullptr_t') for initializer list element type
 auto w = {file, nullptr};

Reply:
`auto w = writer{file, nullptr}` in these situations.

Reply:
Done! Thanks!